packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -233,7 +233,9 @@ const handleEnterKeyOnList = (editor) => {
 };
 
 const Img = styled.img`
+  max-height: 512px;
   max-width: 100%;
+  object-fit: contain;
```

Comment:
When you have `object-fit:contain`, user won't understand where to begin typing for the next line, may be adding a background for image container would help, could you please confirm with UX.